Source/SwiftLintFramework/Rules/CustomRules.swift

```diff
@@ -70,10 +70,17 @@ public struct CustomRules: Rule, ConfigurationProviderRule {
             }
         }
 
-        return configurations.flatMap {
-            validate(file, configuration: $0).filter { eachViolation in
+        return configurations.flatMap { configuration -> [StyleViolation] in
+            let pattern = configuration.regex.pattern
+            let excludingKinds = Array(Set(SyntaxKind.allKinds()).subtracting(configuration.matchKinds))
```

Comment:
Would you prefer `let excludingKinds = SyntaxKind.allKinds().filter { !configuration.matchKinds.contains($0) }`?